packages/core-web/src/styles/markbind.css

```diff
@@ -247,22 +247,34 @@ pre {
     display: flex;
 }
 
-.function-btn-container {
-    display: grid;
-    height: max-content;
-}
-
 button.function-btn {
-    border: none;
     background-color: transparent;
+    border: 1px solid #d7dadc;
+    border-radius: 5px;
     color: darkgray;
     cursor: pointer;
-    display: inline-block;
+    display: none;
+    margin: 0.35rem;
     padding: 0.35rem;
+    position: absolute;
+    right: 0;
     text-align: center;
     white-space: nowrap;
 }
 
+button.function-btn + button.function-btn {
+    right: 36px;
+}
+
+pre:hover > .function-btn-container > button.function-btn {
+    display: block;
+}
+
+.function-btn-container {
+    display: grid;
+    height: max-content;
+}
```

Comment:
Just want to check, can these be removed? I tried removing them and it doesn't seem to affect the buttons.

Reply:
Agreed! I will remove it